README.md

```diff
@@ -150,6 +150,25 @@ To partition the feed (any field of the protobuf is available), the following op
 -format.hash=SamplerAddress,DstAS
 ```
 
+By default, no compression will be enabled when sending data to kafka
+To change the kafka compression type of the producer side configure the following options
+```
+transport.kafka.compression.type=3
+```
+Optionally to disable compression on the producer end configure the below parameter
+```
+transport.kafka.compression.type=0
+```
+
+List of all compression types:
+```
+CompressionNone   CompressionCodec = 0
```

Comment:
Hmm we should keep the lookup. Just need to handle the case where the mapping is not found.

I think we could use [UnmarshalText](https://pkg.go.dev/github.com/Shopify/sarama#CompressionCodec.UnmarshalText)